monitors/CRMonitor.js

```diff
@@ -43,15 +43,15 @@ class CRMonitor {
       this.logger.warn({
         at: "CRMonitor",
         message: "Cannot compute wallet collateralization ratio because price feed returned invalid value",
-        price
+        price: price.toString()
       });
       return;
     }
 
     this.logger.debug({
       at: "CRMonitor",
       message: "Checking wallet collateralization ratios",
-      price
+      price: price.toString()
```

Comment:
price comes in from the price feed as a big number. converting this to a string makes it loggable.